wcf.targets

```diff
@@ -47,4 +47,9 @@
      <WcfToolsOutputPath>$(WcfRootFolder)bin\Wcf\tools\</WcfToolsOutputPath>
   </PropertyGroup>
 
+  <!-- Override ValidateIgnoreReference from Target 'ValidatePackageTargetFramework' in PackageLibs.targets -->
+  <ItemGroup>
+    <ValidateIgnoreReference Include="System.Private.ServiceModel" />
+  </ItemGroup>
+
```

Comment:
It cannot resolve a number of indirect dependencies (listed below) because it could not determine the generation.
Currently I'm setting the PackageTargetFramework to dotnet5.4, these dependencies are all new and not included in the list of assemblies for "5.4" in Generations.json
They should be listed in the section for "5.5" but they are not yet.
The tools package we pull down is already out of date from the file in the [buildtools](https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Generations.json) repo, it looks like they are moving to a new naming convention, and "5.5" will now be "1.4".
Should I open an Issue in the buildtools repo to add these new contracts to "1.4"?

So in the meantime I think we should ignore validation on S.P.SM. 

System.Security.Cryptography.Primitives
System.Security.Principal.Windows
System.Security.Cryptography.X509Certificates
System.Security.Claims
System.Collections.Specialized
System.Collections.NonGeneric
System.Reflection.TypeExtensions
System.Net.Security
System.Xml.XmlDocument
System.Net.WebSockets
System.Net.WebSockets.Client
System.Net.NameResolution
System.Security.Cryptography.Algorithms
System.Reflection.DispatchProxy
System.Net.Http.WinHttpHandler
System.Security.Cryptography.Encoding
System.Runtime.InteropServices.RuntimeInformatio